packages/app-project/src/shared/components/ConnectWithProject/ConnectWithProject.mock.js

```diff
@@ -58,13 +58,6 @@ export const ConnectWithProjectMock = {
         site: 'tumblr.com',
         url: 'https://tumblr.com/test-project',
       },
-      {
-        name: 'Twitter',
-        label: '',
-        path: 'test-project',
-        site: 'twitter.com',
-        url: 'https://twitter.com/test-project',
```

Comment:
Deleting this mock because I've left the translation strings as 'twitter' and just changed the icon in ConnectWithProject.